geppetto-client/js/components/reduxConnector.js

```diff
@@ -0,0 +1,228 @@
+import { connect } from "react-redux";
```

Comment:
Why not name it just index? For the api user it's not important that components are coming from redux or anywhere else